measure/src/main/scala/org/apache/griffin/measure/datasource/connector/DataConnectorFactory.scala

```diff
@@ -91,11 +91,12 @@ object DataConnectorFactory extends Loggable {
                                    ): KafkaStreamingDataConnector  = {
     val KeyType = "key.type"
     val ValueType = "value.type"
-    val config = dcParam.config
+    val config = dcParam.getConfig
     val keyType = config.getOrElse(KeyType, "java.lang.String").toString
     val valueType = config.getOrElse(ValueType, "java.lang.String").toString
-    (getClassTag(keyType), getClassTag(valueType)) match {
-      case (ClassTag(k: Class[String]), ClassTag(v: Class[String])) => {
+
+    (keyType, valueType) match {
+      case ("java.lang.String", "java.lang.String") => {
         KafkaStreamingStringDataConnector(sparkSession, ssc, dcParam, tmstCache, streamingCacheClientOpt)
```

Comment:
I think match with literal string is too strong with limitation, why not match with the class type?

Reply:
the logic here seems assume the key type and value type are both String, or throw the exception. 

And the type erasure makes `case (ClassTag(k: Class[String]), ClassTag(v: Class[String])) => {` already true

Reply:
Or we could move the type checking logic to configuration validation

Reply:
OK, I think you're right, it will be better to match with string here.